components/brave_wallet_ui/page/reducers/page_reducer.ts

```diff
@@ -244,5 +248,14 @@ export const createPageReducer = (initialState: PageState) => {
 
 export const pageSlice = createPageSlice()
 export const pageReducer = pageSlice.reducer
+export const persistedPageReducer = persistVersionedReducer(
+  pageReducer,
+  {
+    key: 'page',
+    version: PERSISTED_STATE_VERSION,
+    blacklist: ['mnemonic']
```

Comment:
can we be extra sure that the private key (in hex or other form), password, and any auth tokens don't get persisted?

it almost seems like it would be safer to use a whitelist of persistable fields rather than a blacklist. that way if a new auth token or something gets added, there's no chance it would accidentally get saved in plaintext.

Reply:
Agreed. I'll switch these to use whitelists

Reply:
Updated to use whitelists